md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/ProcessTemplateAPI.java

```diff
@@ -644,7 +577,6 @@ public void AdjustNextIdsForApply(ProcessTemplate processTemplate) {
                 processTemplate2 = new ProcessTemplate();
                 LOGGER.debug("processTemplatesForNext.get(count)= " + processTemplateForNext.getProcessId());
                 int count_inner = 0;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this local variable name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS00117)